backend/associations/serializers/election_read_only.py

```diff
@@ -18,14 +17,16 @@ def to_representation(self, instance: Choice):
 
         if instance.show_results:
             response["number_of_votes"] = instance.number_of_votes
+            response["number_of_offline_votes"] = instance.number_of_offline_votes
+            response["number_of_online_votes"] = instance.number_of_online_votes

```

Comment:
Why do we need this? In my opinion, this information is on a need to know basis

Reply:
This is useful to give the detail of the votes. This is the way it is done today.
![image](https://user-images.githubusercontent.com/4300678/88783178-c3471c00-d18e-11ea-95c6-70aafe3fd8a2.png)